### PR TITLE
Remove thrust from altitude graph. Add HDOP, VDOP, and speed accuracy to GPS. Fix vibration instance 3 label.

### PR DESCRIPTION
**Pull request by AlexKlimaj**
Remove thrust from altitude graph. Add HDOP, VDOP, and speed accuracy to GPS. Fix vibration instance 3 label.

@dakejahl @dagar 

image
image
image

### CODE REVIEW

**Review by dakejahl — APPROVED** [review]

File: app/plot_app/configured_plots.py [hunk 1]
```diff
@@ -195,12 +195,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.change_dataset('vehicle_global_position')
     data_plot.add_graph(['alt'], colors8[2:3], ['Fused Altitude Estimation'])
     data_plot.change_dataset('position_setpoint_triplet')
-    data_plot.add_circle(['current.alt'], [plot_config['mission_setpoint_color']],
-                         ['Altitude Setpoint'])
-    data_plot.change_dataset(actuator_controls_0.thrust_sp_topic)
-    if actuator_controls_0.thrust_z_neg is not None:
-        data_plot.add_graph([lambda data: ('thrust', actuator_controls_0.thrust_z_neg*100)],
-                            colors8[6:7], ['Thrust [0, 100]'])
+    if 'position_setpoint_triplet' in data and 'current' in data['position_setpoint_triplet']:
```
Comment (bkueng) [inline app/plot_app/configured_plots.py:198]: Why is this changed?

File: app/plot_app/configured_plots.py [hunk 2]
```diff
@@ -195,12 +195,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.change_dataset('vehicle_global_position')
     data_plot.add_graph(['alt'], colors8[2:3], ['Fused Altitude Estimation'])
     data_plot.change_dataset('position_setpoint_triplet')
-    data_plot.add_circle(['current.alt'], [plot_config['mission_setpoint_color']],
-                         ['Altitude Setpoint'])
-    data_plot.change_dataset(actuator_controls_0.thrust_sp_topic)
-    if actuator_controls_0.thrust_z_neg is not None:
```
Comment (bkueng) [inline app/plot_app/configured_plots.py:201]: Can you separate the commit and provide the reason for the change for future reference?
We could also shift the thrust into the range of the altitude instead.

File: app/plot_app/configured_plots.py (hunk 1 above)
Comment (AlexKlimaj) [reply app/plot_app/configured_plots.py:198]: Without this, the Y axis range always starts at 0.

File: app/plot_app/configured_plots.py (hunk 2 above)
Comment (AlexKlimaj) [reply app/plot_app/configured_plots.py:201]: Split the commit and added a comment.

**Review by dagar — APPROVED** [review]

File: app/plot_app/configured_plots.py (hunk 1 above)
Comment (bkueng) [reply app/plot_app/configured_plots.py:199]: Well this isn't correct though. How did you test this?
You probably need to take into consideration the `valid` flag.

Comment (AlexKlimaj) [reply app/plot_app/configured_plots.py:199]: Do you have a log where this doesn't work?

Comment (bkueng) [reply app/plot_app/configured_plots.py:199]: Check any log that has some mission on https://logs.px4.io/browse, eg https://logs.px4.io/plot_app?log=22a7603d-edf4-476a-8b0e-0aa0a73b107b

File: app/plot_app/configured_plots.py (hunk 2 above)
Comment (AlexKlimaj) [reply app/plot_app/configured_plots.py:201]: https://github.com/PX4/flight_review/pull/287
